WowPacketParserModule.V7_0_3_22248/Parsers/GarrisonHandler.cs

```diff
@@ -192,5 +219,140 @@ public static void HandleGetGarrisonInfoResult(Packet packet)
+
+            packet.ReadBit("PlayActivationCinematic");
+        }
+
+        [Parser(Opcode.SMSG_GARRISON_START_MISSION_RESULT)]
+        public static void HandleGarrisonStartMissionResult(Packet packet)
+        {
+            packet.ReadInt32E<MissionResultType>("Result");
+            packet.ReadInt16("SessionMissionCount");
+            ReadGarrisonMission(packet, "Mission");
+
+            var followerCount = packet.ReadInt32("FollowerCount");
+            for (int i = 0; i < followerCount; i++)
+                packet.ReadInt64("FollowerDBIDs", i);
+        }
+
+        [Parser(Opcode.SMSG_GARRISON_FOLLOWER_CHANGED_XP)]
+        public static void HandleGarrisonFollowerChangedXp(Packet packet)
+        {
+            packet.ReadUInt32("XPEarned");
+            packet.ReadUInt32("XPSource");
+            ReadGarrisonFollower(packet, "FollowerBefore");
+            ReadGarrisonFollower(packet, "FollowerAfter");
+        }
+
+        [Parser(Opcode.CMSG_OPEN_MISSION_NPC)]
+        public static void HandleGarrisonNpcGUID(Packet packet)
+        {
+            packet.ReadPackedGuid128("NpcGUID");
+            packet.ReadInt32E<GarrisonFollowerType>("FollowerType"); // Indicates which type of missions
+        }
+
+        [Parser(Opcode.SMSG_GARRISON_OPEN_MISSION_NPC)]
+        public static void HandleGarrisonOpenMissionNpc(Packet packet)
+        {
+            packet.ReadPackedGuid128("NpcGuid");
+            packet.ReadUInt32E<GarrisonFollowerType>("FollowerType"); // Indicates which type of missions
+        }
+
+        [Parser(Opcode.SMSG_GARRISON_LANDING_PAGE_SHIPMENT_INFO)]
+        public static void HandleGarrisonLandingPageShipmentInfo(Packet packet)
+        {
+            packet.ReadUInt32("UnkUInt32");
+            uint shipmentsCount = packet.ReadUInt32("ShipmentsCount");
+            for (uint i = 0; i < shipmentsCount; i++)
+                ReadGarrisonShipment(packet, "Shipment", i);
+        }
+
+        [Parser(Opcode.SMSG_GARRISON_MISSION_BONUS_ROLL_RESULT)]
+        public static void HandleGarrisonMissionBonusRollResult(Packet packet)
+        {
+            ReadGarrisonMission(packet, "Mission");
+            packet.ReadUInt32("MissionRecID");
+            packet.ReadUInt32E<MissionResultType>("Result");
+        }
+
+        [Parser(Opcode.SMSG_GARRISON_COMPLETE_MISSION_RESULT)]
+        public static void HandleGarrisonFinishMission(Packet packet)
+        {
+            packet.ReadUInt32E<MissionResultType>("Result");
+            ReadGarrisonMission(packet, "Mission");
+            packet.ReadUInt32("MissionRecId");
+
+            if (ClientVersion.AddedInVersion(ClientVersionBuild.V7_2_0_24015)) // even earlier?
+            {
+                uint count = packet.ReadUInt32("MissionFollowerCount");
+                for (uint i = 0; i < count; i++)
+                {
+                    packet.ReadUInt64("FollowerDBID", i);
+                    packet.ReadUInt32("UnkUint32", i); // MissionCompleteState ?
```

Comment:
MissionCompleteState <-- ?

Reply:
oh, dk about this one, mostly seen there just zero

Reply:
0 <-- SUCCESS well i keep it open for now ^^